todo_app/data/trello_api.py

```diff
@@ -54,7 +114,7 @@ def get_trello_cards():
         else:
             due_date = card['due']
 
-        card_list.append(TrelloCard(card['id'], card['name'], card['idList'], datetime.datetime.strptime(due_date, '%Y-%m-%dT%H:%M:%S.%fZ'), card['desc']))
+        card_list.append(TrelloCard(card['id'], card['name'], card['idList'], datetime.datetime.strptime(due_date, '%Y-%m-%dT%H:%M:%S.%fZ'), card['desc'], datetime.datetime.strptime(card['dateLastActivity'], '%Y-%m-%dT%H:%M:%S.%fZ').date()))
```

Comment:
Minor, but pulling out the dates, or even just extracting the datetime format here, would help reduce this line length and make it easier to process